app/controllers/supplejack_api/records_controller.rb

```diff
@@ -36,10 +37,6 @@ def index
       else
         render request.format.to_sym => { errors: @search.errors }, status: :bad_request
       end
-    rescue RSolr::Error::Http => e
```

Comment:
These are already being handled in @search.valid? called above.

Reply:
Are you sure we are handling Rsolr::Error in search.valid?

Noice one

Reply:
Yes. Double checked it with Ben.